lms/djangoapps/certificates/api.py

```diff
@@ -20,7 +20,8 @@
 from lms.djangoapps.branding import api as branding_api
 from lms.djangoapps.certificates.generation_handler import \
     generate_user_certificates as _generate_user_certificates
-from lms.djangoapps.certificates.generation_handler import _is_using_certificate_allowlist
+from lms.djangoapps.certificates.generation_handler import \
+    is_using_certificate_allowlist as _is_using_certificate_allowlist
```

Comment:
😅 Follow-up: Does it need to be `as`ed with the `_` here?

Reply:
We don't want __is_using_certificate_allowlist_ to be called by anyone outside of this django app (because there are better methods they should be using instead).